tests/e2e/workloads/amq/test_rgw_kafka_notifications.py

```diff
@@ -25,6 +28,7 @@
 
 @tier1
 @bugzilla("1937187")
+@bugzilla("1958818")
```

Comment:
What is the polarion test id for this? Could you please create one if not done yet?

Reply:
will modify/add the steps to the existing polarion test case